workloads/kube-burner/common.sh

```diff
@@ -177,3 +177,122 @@ remove_update_taint() {
   # This is only here until BZ https://bugzilla.redhat.com/show_bug.cgi?id=2035005 gets resolved
   oc adm taint nodes UpdateInProgress:PreferNoSchedule- --all || true
 }
+
+
+label_node_label() {
```

Comment:
All this new code is only used by the concurrent builds test, I think we should move it to a new *.sh file and source that file from run.sh, in order to prevent the number of lines of common.sh growing too much.